CMakeLists.txt

```diff
@@ -17,7 +17,7 @@
 #============================================================================
 
 cmake_minimum_required(VERSION 3.1)
-set(CMAKE_OSX_DEPLOYMENT_TARGET "10.9" CACHE STRING "Minimum OS X deployment version")
+set(CMAKE_OSX_DEPLOYMENT_TARGET "11.0" CACHE STRING "Minimum OS X deployment version")
```

Comment:
This _should_ not be needed. Xcode _should_ update the target to the minimum supported version (11.0 for ARM64) automatically.